draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -6437,8 +6442,9 @@ WWW-Authenticate: Basic realm="simple", Newauth realm="apps",
 <t>
    Most of these header fields, where indicated, define a wildcard value ("*")
    to select unspecified values. If no wildcard is present, values that are
-   not explicitly mentioned in the field are considered unacceptable, except
-   for within <x:ref>Vary</x:ref> where it means the variance is unlimited.
+   not explicitly mentioned in the field are considered unacceptable.
+   Within <x:ref>Vary</x:ref>, the wildcard value means that the variance
+   is unlimited.
```

Comment:
Please check this one; I was inferring that the "except" matches up to the "most" in the previous sentence.